NetworkPkg/Dhcp4Dxe/Dhcp4Impl.c

```diff
@@ -824,7 +824,15 @@ EfiDhcp4Start (
 
   if (CompletionEvent == NULL) {
     while (DhcpSb->IoStatus == EFI_ALREADY_STARTED) {
-      DhcpSb->UdpIo->Protocol.Udp4->Poll (DhcpSb->UdpIo->Protocol.Udp4);
+      // MU_CHANGE [BEGIN] - Add Udp4 Polling break support
+      Status = DhcpSb->UdpIo->Protocol.Udp4->Poll (DhcpSb->UdpIo->Protocol.Udp4);
+      if (EFI_ERROR (Status)) {

```

Comment:
Is the intention to catch things like `EFI_DEIVCE_ERROR`? Should this have an exception for `EFI_TIMEOUT`?